dev/beaker/10_install_puppet.rb

```diff
@@ -0,0 +1,6 @@
+test_name "Install puppet on target machine"
+
+puppet_version = "3.7.5"
```

Comment:
No, I honestly forgot I left it as 3.7. It worked and I said "cool :+1:" and kept going. I can change it to puppet 4 if you'd like.